source/SimplePicker.js

```diff
@@ -171,6 +171,7 @@
 				this.animateChanged();
```

Comment:
Nevermind. I just tried it and it doesn't actually inhibit the animation. It would need to be slower than even an asyncMethod...